libnl-route-3.sym

```diff
@@ -843,10 +843,14 @@ global:
 	rtnl_class_hfsc_set_usc;
 	rtnl_link_inet6_addrgenmode2str;
 	rtnl_link_inet6_get_addr_gen_mode;
+	rtnl_link_inet6_get_flags;
```

Comment:
a symbol version section in the linker script cannot ever be modified after a release is out.

Actually, 3.3.0-rc1 is already out. I would prefer not to add new API after the release-candidate. Can we target 3.4.0 (with linker section "libnl_3_4")?